modules/src/erc20.rs

```diff
@@ -0,0 +1,301 @@
+use odra::{
+    contract_env,
+    types::{event::OdraEvent, Address, Balance},
+    Mapping, Variable
+};
+
+use self::{
+    errors::Error,
+    events::{Approval, Transfer}
+};
+
+#[odra::module]
+pub struct Erc20 {
+    decimals: Variable<u8>,
+    symbol: Variable<String>,
+    name: Variable<String>,
+    total_supply: Variable<Balance>,
```

Comment:
```suggestion
    total_supply: Variable<U256>,
```
The type for balance must to be `U256`. Reasons:
- we need to be as close to original standard as possible,
- this will be used by ERC20-based assets biridged into Casper.